From f4bf9a0989a99aa7a43f7af3ea34784a1b587e21 Mon Sep 17 00:00:00 2001
From: The-going <48602507+The-going@users.noreply.github.com>
Date: Sun, 24 Jul 2022 13:56:50 +0300
Subject: [PATCH] arm64: dts: sun50i-h6-orangepi.dtsi: Rollback r_rsb to r_i2c

This fix affects two boards:
sun50i-h6-orangepi-lite2.dts
sun50i-h6-orangepi-one-plus.dts

It also depends on the revision of the board.
If your board fails when working with this fix, just disable
it by adding (-) minus to the first position of the line
in which the patch is written in the series.conf file.
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
index ee1919b80..15f4d7de9 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi.dtsi
@@ -129,12 +129,12 @@ &r_pio {
 	vcc-pm-supply = <&reg_bldo3>;
 };
 
-&r_rsb {
+&r_i2c {
 	status = "okay";
 
-	axp805: pmic@745 {
+	axp805: pmic@36 {
 		compatible = "x-powers,axp805", "x-powers,axp806";
-		reg = <0x745>;
+		reg = <0x36>;
 		interrupt-parent = <&r_intc>;
 		interrupts = <GIC_SPI 96 IRQ_TYPE_LEVEL_LOW>;
 		interrupt-controller;
-- 
2.35.3

